### DISCUSSION TITLE
How do I deal with my app version number being changed?

### DISCUSSION THREAD

**Discussion by rectorsquid** [microsoft/winget-pkgs#25375] [Q&A] [answered]
I created a manifest and submitted it a while ago. I see the version number of the app as part of the path to the manifest on my local system and I think that the version number is associated with the app in a rather tight fashion. I just now updated my app and it has a new version number. Do I create a new manifest and submit it? Or do I try to do an update and that will deal with the version number change?

**Comment by ImJoakim** [comment, marked as answer]
Here's 2 scenarios (I'll be using Jackett and Google Chrome as examples):

1. You have version `0.18.537.0` where the `InstallerURL` is a download directly to that `0.18.537.0` version. Example: https://github.com/Jackett/Jackett/releases/download/v0.18.537/Jackett.Installer.Windows.exe
If this is the case, you would upload a new manifest with a new version number as the folder and `InstallerURL`, making it so you have 2 versions of the application listed. For this application, we now have multiple versions listed because there is a direct download to a specific version.

![image](https://user-images.githubusercontent.com/52826317/129147656-99934123-1bf6-43e7-8c95-6a6d1f386404.png)

2. The `InstallerURL` is the exact same and only installs the latest version. Example: https://dl.google.com/edgedl/chrome/install/GoogleChromeStandaloneEnterprise64.msi
In this scenario, you would have to replace the current manifest and update the version in the already listed manifest. For this application, we only have one version listed because the `InstallerURL` only downloads the latest version and not a specific version.

![image](https://user-images.githubusercontent.com/52826317/129147699-1d858fb2-1cf3-4170-b221-05a26d543446.png)

Hope this helps 😃

**Reply by ImJoakim** [reply]
Also, @denelon Are you able to move this discussion to [winget-pkgs/discussions](https://github.com/microsoft/winget-pkgs/discussions)?
